Applications/spire/source/charting/charting_window.cpp

```diff
@@ -146,21 +144,16 @@ ChartingWindow::ChartingWindow(Ref<SecurityInputModel> input_model,
   m_security_widget->setFocus();
 }
 
-void ChartingWindow::set_model(std::shared_ptr<ChartModel> model) {
-  m_model = model;
+void ChartingWindow::set_models(std::shared_ptr<ChartModel> chart_model,
+    std::shared_ptr<TechnicalsModel> technicals_model) {
+  m_model = chart_model;
+  m_technicals_model = technicals_model;
```

Comment:
At what point is this model getting loaded?

Reply:
It gets loaded in charting_ui_tester's main.cpp file.